### ISSUE TITLE
Inconsistent music reset after getting a life

### ISSUE THREAD

**Issue by AlexKhayrullin** [overbound/SonicTimeTwisted#162]
After getting a life through an item box, the act music continues after the jingle is over.
After getting a life by collecting 100 rings, the act music resets

**Comment by nkrapivin** [comment]
Can't reproduce on latest release/1.1 branch:
```gml

if (keyboard_check_pressed(vk_f6)) {
    // icon box
    player_get_lives(1);
}
else if (keyboard_check_pressed(vk_f7)) {
    // rings, press once again after ring theme stops.
    player_get_rings(99);
}
```

This code results in the same thing, the act music resumes from where it was paused.

**Comment by nkrapivin** [comment]
A fix is attempted in #198

**Comment by nkrapivin** [comment]
Can this be closed if the issue's fixed?
